rules/integrations/o365/persistence_microsoft_365_global_administrator_role_assign.toml

```diff
@@ -0,0 +1,56 @@
+[metadata]
+creation_date = "2022/01/06"
+integration = "o365"
+maturity = "production"
+updated_date = "2022/01/06"
+
+[rule]
+author = ["Elastic"]
+description = """
+In Azure Active Directory (Azure AD), permissions to manage resources are assigned using Roles. The Global Administrator
+is a role that enables users to have access to all administrative features in Azure AD and services that use Azure
+Active Directory identities like the Microsoft 365 Defender portal, the Microsoft 365 compliance center, Exchange,
+SharePoint Online, and Skype for Business Online. Attackers can add users as Global Administrators to maintain access
+and manage all subscriptions and their settings and resources.
+"""
+from = "now-25m"
+index = ["filebeat-*", "logs-azure*"]
+language = "kuery"
+license = "Elastic License v2"
+name = "Microsoft 365 Global Administrator Role Assigned"
+note = """## Config
+
+The Microsoft 365 Fleet integration, Filebeat module, or similarly structured data is required to be compatible with this rule."""
+references = [
+    "https://docs.microsoft.com/en-us/azure/active-directory/roles/permissions-reference#global-administrator"
+]
+risk_score = 47
+rule_id = "88671231-6626-4e1b-abb7-6e361a171fbb"
+severity = "medium"
+tags = ["Elastic", "Cloud", "Microsoft 365", "Continuous Monitoring", "SecOps", "Identity and Access"]
+timestamp_override = "event.ingested"
+type = "query"
+
+query = '''
+event.dataset:o365.audit and event.code:"AzureActiveDirectory" and event.action: "Add member to role." and
+o365.audit.ModifiedProperties.Role_DisplayName.NewValue: "Global Administrator"
```

Comment:
nit:

```suggestion
event.dataset:o365.audit and event.code:"AzureActiveDirectory" and event.action:"Add member to role." and
o365.audit.ModifiedProperties.Role_DisplayName.NewValue:"Global Administrator"
```

Also this `NewValue` does not include quotes like the azure rule - is that intentional?

Reply:
Yeah, the azure value contains quotes:
![image](https://user-images.githubusercontent.com/26856693/151361886-0b16efd1-a0be-45e4-8e22-44d9681dc667.png)
While this one don't